torchgen/gen.py

```diff
@@ -2076,12 +2086,38 @@ def gen_backend_select() -> Dict[str, List[str]]:
     schema_selector = selector
     if force_schema_registration:
         schema_selector = SelectiveBuilder.get_nop_selector()
+    schema_static_template = CodeTemplate(
```

Comment:
nit: we try to stop using `CodeTemplate` in the codegen, and use `f-strings` instead (the existing `CodeTemplate` usages should just be removed at some point)